controllers/complianceeventsapi/server.go

```diff
@@ -454,31 +619,34 @@ func getComplianceEvents(db *sql.DB, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	query := fmt.Sprintf(`%s
+	whereClause, filterValues := getWhereClause(queryArgs)
+
+	query := fmt.Sprintf(`%s%s
```

Comment:
Can we add an example of what the first part of this query looks like? It would've been nice if the WHERE could be in the string below, but I think that isn't possible because that whole part is optional. But as a result, it's slightly harder to see what the query will look like when it's put together.